orchestrator/api/heartbeat.py

```
@@ -0,0 +1,201 @@
+            "created_at": r.created_at.isoformat() if r.created_at else None,
+        }
+        for r in rows
+    ]
+
+
+# ── Agent Heartbeat ────────────────────────────────────────────────
+
+@router.post("/agents/{agent_id}/run")
+async def run_agent_heartbeat(
+    agent_id: int,
+    ctx: RequestContext = Depends(get_request_context_hybrid),
```

Ruff (0.15.0)

[warning] Unused function argument: `ctx`

(ARG001)